gulp/build.js

```diff
@@ -1,8 +0,0 @@
-'use strict';
-
-var gulp = require('gulp'),
-    runSequence = require('run-sequence');
-
-gulp.task('build', function(callback) {
```

Comment:
`gulp build` is currently required to create the release build and deploy the site on AWS. Is there something equivalent that can be used now?

Reply:
See https://github.com/Mr-Martinez/r6-operator-counters/blob/upgrade-gulp-4.0/buildspec.yml

This is the build pipeline which performs `gulp build` and then copies the "dist" folder to host the site.

Reply:
I can't find anything about this in the Gulp docs, slightly new.

Reply:
I think it was a custom defined task in the build.js file that has been removed. It just happened to be called `gulp build`, I don't think the name is important. `npm run start` is good for testing changes and developing but we need something to be able to produce a built folder with all the images, static HTML etc. `npm run start` can't be used for that.

Thanks for looking into this!

Reply:
I added a line of code and it should do the trick.

I ran 'gulp build' directly on cli and it ran.

I also added gulp-cli to dev dependencies, although I downloaded it globally first so I'm not confident it will work for new installs to test out 'gulp build'.

Can you see if this does the trick?

Reply:
Added 1 more comment, I think it is almost there!